orchestration/SetupSubBaskets.py

```diff
@@ -36,14 +32,15 @@ def run(self):
                 'subBasketDir': subBasketDir
             }
 
-            os.makedirs(subBasketDir)
+            Path(subBasketDir).mkdir(parents=True, exist_ok=True)
             for product in pair['products']:
                 srcPath = os.path.join(self.inputLocation, product)
                 destPath = os.path.join(subBasketDir, product)
                 # Creates a symlink to the original file, resolves the symlink if 
                 # it is one so not chaining so we can mount the correct data 
                 # directories more easily
-                Path(destPath).symlink_to(Path(srcPath).resolve())
+                if not Path(destPath).exists():
+                    Path(destPath).symlink_to(Path(srcPath).resolve())
```

Comment:
Make the orchestration rerunnable by making the basket setup not error if files already exist.